src/mem_functions.h

```diff
@@ -7,7 +7,7 @@ extern "C" {
 void* memset(void*, int, size_t);
 void* memcpy(void* dest, const void* src, size_t n);
 int strcmp(const char* str1, const char* str2);
-void* memmove(void* to, const void* from, size_t numBytes);
+void* memmove(void* dst, const void* src, size_t len);
```

Comment:
tiny nit but should it match memcpy if we're changing it anyway?

Reply:
Fixed